### ISSUE TITLE
windows10: input shift+"

### ISSUE THREAD

**Issue by volcanoxj** [PowerShell/PSReadLine#996]
### Environment                                                                                                                                                                                                                                                                 PSReadLine: 2.0.0-beta4
PowerShell: 7.0.0-preview.2
OS: Microsoft Windows 10.0.18362
Last 7 Keys
```
 " Enter
 Enter
 l s Enter
 "
```

### Exception
```
System.ArgumentOutOfRangeException: StartIndex cannot be less than zero.
Parameter name: startIndex
   at System.String.Substring(Int32 startIndex, Int32 length)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Render()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(PSKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by msftrncs** [comment]
This was possibly fixed in PR #967, now part of beta5.  Looks like it might have been related to the length of the prompt.

**Comment by daxian-dbw** [comment]
I agree with @msftrncs that this might be fixed in #967. Please upgrade to `2.0.0-beta5` from PowerShellGallery and try it out.
